qa/colobus/config

```diff
@@ -557,35 +515,30 @@ group php.qa {
 group php.qa.reports {
   num => 75
   path => /var/spool/mlmmj/qa-reports
-  mail => qa-reports@lists.php.net
   desc => make test QA reports.
 }
 
 group php.smarty.cvs {
   num => 76
   path => /var/spool/mlmmj/smarty-cvs
   desc => Automated mailings from commits to the Smarty CVS repository
-  followup => smarty.dev
```

Comment:
This looks wrong? AFAIK it is used to set the reply-to to the smart.dev list?

Reply:
colobus shows a group as 'active' if followup is set without checking that the followup group is also active. Rather than changing the behavior of colobus, it's easier to just remove the followup from the config for groups that should be inactive.